.gitignore

```diff
@@ -44,12 +45,6 @@ deps/dpdk*
 # OSX
 .DS_Store
 
-# Protobuf
```

Comment:
At first I thought I needed to keep empty sub-directories (`proto/`), and `.gitignore` would make them not-quite-empty. But I needed the `__init__.py` files anyway, so it turned out I didn't need `.gitignore` in each directory after all.